WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -186,9 +186,7 @@ final class ProductFormViewController<ViewModel: ProductFormViewModelProtocol>:
     }
 
     @objc func publishProduct() {
-        if viewModel.formType == .add {
-            ServiceLocator.analytics.track(.addProductPublishTapped, withProperties: ["product_type": product.productType.rawValue])
-        }
+        ServiceLocator.analytics.track(.addProductPublishTapped, withProperties: ["product_type": product.productType.rawValue])
```

Comment:
I wonder if we shouldn't change this behavior. While what you say makes sense, all of our measurements and baselines already have the original frequency of this event factored in. So I think it's best if we can avoid changing it, does that make sense?

Reply:
Agree! Reverted `addProductPublishTapped` change.